client/src/app/site/motions/components/motion-import-list/motion-import-list.component.html

```diff
@@ -47,6 +47,14 @@
                 </mat-option>
             </mat-select>
         </mat-form-field>
+        <mat-form-field>
+            <mat-label translate> Text separator</mat-label>
```

Comment:
one whitespace too much.
(this might translate ` Text seperator` with additional whitespace at the beginning